app/javascript/components/pages/CityShow.js

```diff
@@ -4,58 +4,160 @@ import { Link } from "react-router-dom";
 import cityData from "../TicityCities.js";
 
 class CityShow extends React.Component {
-  render() {
-    const city = Object.values(cityData).filter(
-      (x) => x.name === "San Diego"
+  componentDidMount() {
+    const { search } = this.props;
+    let currentCity = this.props.currentCity;
+
+    const focusCity = Object.values(cityData).filter(
```

Comment:
Optional: Array.find will could be dropped in here and negate the need to use `[0]` at the end of this.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
changed/validated that line of code to be:
const focusCity = Object.values(cityData).find(
      (x) => x.name === currentCity
    );